### DISCUSSION TITLE
Tailwindcss 3.3.0 and later cannot be used in node16

### DISCUSSION THREAD

**Discussion by fengzai6** [tailwindlabs/tailwindcss#13965] [Help]
On this Monday, when I ran `yarn,` I reported this error:
` glob@10.4.3: The engine "node" is incompatible with this module. Expected version "&gt; =18". Got "16.20.2'`

Excluding knowing that tailwind uses this dependency, I can only use Tailwind version 3.2.7 and other lower versions at present
![image](https://github.com/tailwindlabs/tailwindcss/assets/112751823/7380def4-12b9-4deb-952f-f28eb13dd6c9)

**Comment by andrew2764** [comment]
The error is probably caused by [this commit](https://github.com/isaacs/node-glob/commit/8a69def3cad0de9ba26ca831065ffe448d153de3) 3 days ago to the `glob` repo. It looks like glob@10.4.3 no longer supports node v16 and several people including myself started seeing this same error. 

Simply changing my node version to 18 solved this for me but it may not be that straightforward for everyone
